### ISSUE TITLE
VideoComposer merged with Modelscope

### ISSUE THREAD

**Issue by tin2tin** [huggingface/diffusers#4611]
### Model/Pipeline/Scheduler description

This is very good news, since VideoComposer comes with multiple new features for ex. img2vid and vid2vid. With this merge, maybe open source txt2vid can be shoulder to shoulder with the closed source solutions. 

Examples:
https://videocomposer.github.io/

![image](https://github.com/huggingface/diffusers/assets/1322593/6423a3c2-b565-4ebc-9cfe-d2e6f9a48073)

### Open source status

- [ ] The model implementation is available
- [ ] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation

https://github.com/modelscope/modelscope/pull/431
https://github.com/damo-vilab/videocomposer

**Comment by sayakpaul** [comment]
We're open to accepting PRs for this!

**Comment by tin2tin** [comment]
Image2Video: https://modelscope.cn/models/damo/Image-to-Video/summary

Video2Video: https://modelscope.cn/models/damo/Video-to-Video/summary

Demo: https://modelscope.cn/studios/damo/I2VGen-XL-Demo/summary

**Comment by patrickvonplaten** [comment]
What is your feeling about the quality here @tin2tin ? Do you think it's really good?

**Comment by tin2tin** [comment]
Camenduru has added this, so it is easy to test: https://openxlab.org.cn/apps/detail/camenduru/Image2Video
You can find our community videos in his text to video discord: https://discord.com/channels/1077510466470952990/1110003274960097331
People seems quite happy about it. The catch is that the model is very big and needs 20 GB+ of VRAM as it is. But it's properly 32 bit. 
I've implemented the Diffusers vid2vid in my Blender add-on and added a img2vid version of it, and it is pretty good, but quite static. The strength value seems to both cover motion and fabulating on the input material, so if you want more motion, you'll end up with something which doesn't look like the input. 
Looking at the VideoComposer options, they're looking quite strong, properly stronger than the AnimateDiff alternative. But it's very hard for me to say which one is best, but the community is for sure very eager to get a better img2vid and vid2vid open source solution. 
(Btw. there should be coming and improved model out soon, which might replace Zeroscope).
TL DR; ...I don't know.

**Comment by patrickvonplaten** [comment]
Hmm yeah it does look really interesting! @sayakpaul @yiyixuxu @patil-suraj @williamberman any thoughts here?

**Comment by sayakpaul** [comment]
The download stats aren't very promising yet for the video checkpoint (even the ones that don't suffer from the watermarking problem). But I'd say we go for it given the dynamics.
